features/FxA-54-push-on-status-change/README.md

```diff
@@ -63,9 +63,14 @@ in response to the following actions:
 * User deletes their account
 * User resets their password
 
+When the fxa-auth-server receives a `410 Gone`
```

Comment:
:+1: to clobbering the push fields in the database.

Note that they need to be set to the empty string rather `null`, because the `updateDevice` stored procedure uses `COALESCE` on the inputs.